fast_build_and_deploy_docs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+latesttag=$(git describe --tags `git rev-list --tags --max-count=1`)
+echo checking out ${latesttag}
+git checkout ${latesttag}
+git submodule update --remote
```

Comment:
did you try running `fast_build_and_deploy_docs.sh`? I think all these can be removed - if you do a fast doc build, you would do it with your local changes, not with the latest tag